server/routes/admin/categories/categories.js

```diff
@@ -29,8 +32,10 @@ router.post('/add', function (req, res, next) {
 router.get('/edit/:categoryId', (req, res) => {
     const {categoryId} = req.params;
     const callback = (error, category) => {
-        res.render("admin-edit-category", {
-            category: category
+        res.render("admin-edit-and-add-category", {
+            category: category,
```

Comment:
don't be lazy with variable names - `desTitle` is a bad name, just say `descriptionTitle`

Reply:
you should not pass these "default" values to the View. Just use input `placeholder` to display this information.

Reply:
do you mean I do not need labels?

Reply:
I did that just to be logic title for add or edit